test/setup.js

```diff
@@ -200,14 +208,14 @@ function extractImportantFields (parsedTorrent) {
 }
 
 function copy (pathFrom, pathTo) {
+  const cpFile = require('cp-file')
   try {
-    fs.copySync(pathFrom, pathTo)
```

Comment:
Currently we don't.

I don't think we should do lazy requires in the integration tests. Startup time isn't an issue here.